recipes/openquake/meta.yaml

```diff
@@ -0,0 +1,272 @@
+{% set version = "2.8.0" %}
+
+package:
+  name: openquake.engine
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/o/openquake.engine/openquake.engine-{{ version }}.tar.gz
+  sha256: 55aed57bb65395e733106e5a1b2cb3fde9d8966f40f3b02167ccc9446d356473
+
+build:
+  number: 0
+  skip: True  # [py2k]
```

Comment:
Even though [setup.py](https://github.com/gem/oq-engine/blob/master/setup.py#L105) mentions that this works with Python 2 it does not. `openquake` imports Python 3.5 only modules.